docs/source/nodes_and_pipelines/pipeline_introduction.md

```diff
@@ -252,3 +249,131 @@ Nodes that are created with input or output names that contain `.` risk a discon
 This is because `.` has a special meaning internally and indicates a namespace pipeline. In the example, the outputs segment should be disconnected as the name implies there is an "output1" namespace pipeline. The input is not namespaced, but the output is via its dot notation. This leads to Kedro processing each separately. For this example, a better approach would've been writing both as `input1_kedro` and `output1_kedro`.
 
 We recommend use of characters like `_` instead of `.` as name separators.
+
+
+## How to create a new blank pipeline using the `kedro pipeline create` command
+
+When managing your Kedro project, we recommend grouping related tasks into individual pipelines to achieve modularity. A project typically contains many tasks, and organising frequently executed tasks together into separate pipelines helps maintain order and efficiency. Each pipeline should ideally be organised in its own folder, promoting easy copying and reuse within and between projects. Simply put: one pipeline, one folder. To assist with this, you can use the following command (the pipeline name must adhere to [Python naming conventions](https://realpython.com/python-pep8/#naming-conventions)):
```

Comment:
From here I'd direct readers to the next page, explaining Modular Pipelines (see review comment)

```suggestion
When managing your Kedro project, we recommend grouping related tasks into individual pipelines to achieve modularity. A project typically contains many tasks, and organising frequently executed tasks together into separate pipelines helps maintain order and efficiency. Each pipeline should ideally be organised in its own folder, promoting easy copying and reuse within and between projects. Simply put: one pipeline, one folder. To assist with this, Kedro introduces the concept of [Modular Pipelines](./modular_pipelines), which are described in the next section.
```